internal/core/operations/definition.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/topfreegames/maestro/internal/core/entities/operation"
 )
 
+type DefinitionConstructor func() Definition
+
+func NewDefinitionConstructors() map[string]DefinitionConstructor {
```

Comment:
Do you think defining a file for those "DefinitionConstructors" would make it easier to read and enable easier changes?

We would initially create a type alias:
```go
type DefintionConstructors map[string]DefinitionConstructor

func NewDefinitionConstructors() DefinitionConstructors {
	return DefintionConstructors(map[string]DefintionConstructor)
}
```

And other places that rely on it would just reference it as `DefintionConstructors`.